README.rst

```diff
@@ -459,6 +459,10 @@ Additional resources
 --------------------
 
 - GPU Technology Conference 2018 presentation about DALI, T. Gale, S. Layton and P. Tredak: `slides <http://on-demand.gputechconf.com/gtc/2018/presentation/s8906-fast-data-pipelines-for-deep-learning-training.pdf>`_, `recording <http://on-demand.gputechconf.com/gtc/2018/video/S8906/>`_.
+- GPU Technology Conference 2019 presentation about DALI, J.Lisiecki and Michał Zientkiewicz: `slides <https://developer.download.nvidia.com/video/gputechconf/gtc/2019/presentation/s9925-fast-ai-data-pre-processing-with-nvidia-dali.pdf>`_, `recording <https://developer.nvidia.com/gtc/2019/video/S9925/video>`_.
```

Comment:
I would suggest adding the titles of the presentations, to be less generic.

Reply:
Done